loom/common/platform/platformMobileIOS.mm

```diff
@@ -337,35 +369,64 @@ bool platform_wasOpenedViaRemoteNotification()
 ///checks if a given sensor is supported on this hardware
 bool platform_isSensorSupported(int sensor)
 {
-    ///TODO: 1844: Support sensors on iOS
-    return false;
+    BOOL supported = false;
+    
+    switch (sensor)
+    {
+        case 0:
```

Comment:
Yeah, figured, also asked this on #497, would be probably good to have one on the cpp side, but it's ok if it's too much trouble.